packages/aggregator/src/analytics/aggregateAnalytics/aggregateAnalytics.js

```diff
@@ -113,6 +114,8 @@ export const aggregateAnalytics = (
       return countPerPeriodPerOrgGroup(analytics, aggregationConfig);
     case AGGREGATION_TYPES.REPLACE_ORG_UNIT_WITH_ORG_GROUP:
       return replaceOrgUnitWithOrgGroup(analytics, aggregationConfig);
+    case AGGREGATION_TYPES.YEAR_ON_YEAR:
```

Comment:
Pretty sure this was and old attempt, we did it all in web-config-server in the end right?